src/trade/tradeConf.es6

```diff
@@ -220,15 +200,6 @@ export const init = (data, extra, show_callback, hide_callback) => {
       },
       ticks: {
          array: [],
-         average: () => {
-            const ticks = state.ticks;
-            const array = ticks.array;
-            let sum = 0;
-            for(let i = 0; i < array.length; ++i)
-               sum += array[i].quote*1;
-            const avg = sum / (array.length || 1);
-            return avg;
-         },
```

Comment:
@4p00rv My guess is that "tick.barrier" will return the new barrier value for Asians. Make sure that's the case.